### ISSUE TITLE
CSV Upload: Support existing use cases

### ISSUE THREAD

**Issue by joeflack4** [jhu-bids/TermHub#247]
## Overview
Support existing users trying to upload.

## To support
- [ ] Adit / Richard Moffit's group
- [ ] Bryan Laraway

## Additional tasks
- [ ] Add variations of these as test cases

**Comment by Sigfried** [comment]
@joeflack4: I think we lost track of this one. Did that happen because it depends on the objects API stuff?

**Comment by joeflack4** [comment]
I don't think it depends on that. I think it just got deprioritized behind that. I personally wanted to keep the current functionality of uploading (a) a version separately before the refactor to have it figure out if it is (b) uploading versions &/or containers. If you want, we could probably without too much work re-enable 'a'. I can also work on this again after we finish w/ the objects DB sync.
